microprofile-metrics/pom.xml

```diff
@@ -9,8 +9,33 @@
     </parent>
 
     <artifactId>microprofile-metrics</artifactId>
-    
-    <dependencies>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.microprofile</groupId>
+            <artifactId>microprofile</artifactId>
+            <type>pom</type>
+            <scope>provided</scope>
```

Comment:
We shouldn't duplicate specifying scope in `dependencyManagement` and then in `dependencies` in each submodule. I've tried to google some examples and best practices but did not find definitive answer. 

But based on discussion in stackoverflow[1] i'm more for defining scope in submodules. wdyt?

[1] https://stackoverflow.com/questions/15221299/dependencymanagement-and-scope

Reply:
I added scopes to root pom.xml intentionally

 - people will probably copy those deps to their new module, throw away just version line, they won't think about defining scope
 - previously I did some deps analysis based on dependencyManagement definition and missing scope definition was big trouble for that analysis so I tend to keep scope there too